addons/mail/wizard/mail_template_preview_view.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <!-- Email Template Preview -->
+        <record model="ir.ui.view" id="mail_template_preview_view_form">
```

Comment:
Almost, just need to invert `model` and `id` now ^^

So: `<record id="mail_template_preview_view_form" model="ir.ui.view">`